js/gateio.js

```diff
@@ -2534,6 +2619,7 @@ module.exports = class gateio extends Exchange {
             throw new ArgumentsRequired (this.id + ' fetchOrder() requires a symbol argument');
         }
         await this.loadMarkets ();
+        const isStop = this.safeValue (params, 'isStop', false);
```

Comment:
The only thing that i'm not entirely sure of – is this `isStop` flag, the naming seems too arbitrary and exchange-specific. I'll think of how to address it.

Reply:
Yeah but we can't use `type` because that refers to the market type

Reply:
We should not send it to the exchange, let's use the following:

```JavaScript
//                      exchange specific ↓               ↓ unified
//                                        ↓               ↓
const stop = this.safeValue2 (params, 'is_stop_order', 'stop', false);
params = this.omit (params, [ 'is_stop_order', 'stop' ]);
```